### DISCUSSION TITLE
Jasco 24770/ZW4006 driver does not track motion - possibly monitoring the wrong endpoint?

### DISCUSSION THREAD

**Discussion by nmpu** [zwave-js/zwave-js#6537] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
I have multiple Jasco 24770/ZW4006 devices that were previously reporting motion under SmartThings. I have tried exclude/include multiple times with identical results on every device. The associated 'motion_detection' sensor entity does not update. I can see motion events in the Z-Wave JS debug window. The switch will automatically turn on with motion if so configured. I also have the similar Jasco 26931/ZW4006 which works as expected. I tried triggering directly on command class 113-1. That works for a while, but eventually stops triggering. At some point (including restart), HA starts reporting errors that <node>-113-1 does not exist. It sure looks like something is getting confused in the driver.

What did you expect to happen instead?
I expect the motion_detection sensor entity to update with motion-- subject to the device configuration parameters.

  1. Z-Wave include a Jasco 24770/ZW4006 device. This should also reset all configuration.
  2. View the device details in HA.
  3. Observe that motion is not detected.

Anything else we should know?
Looking at the Z-Wave JS debug window, the 24770 reports on endpoint 1, while the 26931 reports on endpoint 2. At one point while editing entity names, a _second_ motion_detection entity appeared on a 24770 node and it was working! That was very short-lived.

When I attempt to edit a stalled 113-1 trigger, I see something like:

`Message malformed: Value 70-113-1-Home Security-Motion sensor status can't be found on node Node(node_id=70)`

### Software versions

Driver (node-zwave-js): 12.3.0

Z-Wave JS UI: 9.3.2

### Device information

Manufacturer: Jasco
Model name: 24770/ZW4006
Node ID: 74

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[node_74.json](https://github.com/zwave-js/node-zwave-js/files/13537850/node_74.json)
[interview_node_74.log](https://github.com/zwave-js/node-zwave-js/files/13537851/interview_node_74.log)

[a comment by an automated account is not shown]

**Reply by nmpu** [reply]
Here's my work-around automation which works for a while, but ultimately stalls. It's also disabled with a 'not found' error on restart.

```
trigger:
  - platform: device
    device_id: 9b060788ee2f612e62b91be4ac70b930
    domain: zwave_js
    type: zwave_js.value_updated.value
    command_class: 113
    property: Home Security
    property_key: Motion sensor status
    endpoint: 1
    to: 8
action:
  - service: system_log.write
    data:
      message: "{{ trigger.description }}"
mode: single
```
![automation](https://github.com/zwave-js/node-zwave-js/assets/8411598/3aae47e7-e7cd-4b10-8b41-f5965d0454f6)

Here's a sample event:
![event](https://github.com/zwave-js/node-zwave-js/assets/8411598/2fe06c53-8844-459a-b5a1-18bc5cea5457)

Here's the duplicate motion sensor:
![duplicate_motion](https://github.com/zwave-js/node-zwave-js/assets/8411598/6edd1d49-84d4-44a3-9323-13ecdbbc797f)

And later after a restart:
![duplicate_motion_after_restart](https://github.com/zwave-js/node-zwave-js/assets/8411598/d83329cb-a866-46e0-babf-d2417df13e9d)

**Comment by nmpu** [comment]
Here are sample reports for **24770** (broken) and **26931** (working). The only difference is the endpoint.

[24770_motion_report.log](https://github.com/zwave-js/node-zwave-js/files/14002698/24770_motion_report.log)
[26931_motion_report.log](https://github.com/zwave-js/node-zwave-js/files/14002699/26931_motion_report.log)

**Comment by nmpu** [comment]
Any love for my **Jasco 24770** devices? I'd be perfectly happy triggering off the raw Z-Wave motion messages, but the **HA** integration won't let me because they "can't be found".

_There is strict validation in place based on all known event types, so if you come across an event type that isn’t supported, please open a GitHub issue in the home-assistant/core repository._

**Reply by AlCalzone** [reply]
@nmpu I just came back from 4 months parental leave last week, so there's quite a bit of backlog I have to work through. I'll get to this eventually.

**Reply by nmpu** [reply]
I know, that's why I bumped. Welcome back.

**Reply by nmpu** [reply]
There were config file changes earlier this year which resulted in new endpoints for certain events. This broke some of my existing low-level event handlers for other devices. As a positive side effect, the 24770 motion events are now coerced to endpoint 0 like all the other Jasco motion devices. For the time being, my original issue has been resolved.
